src/components/Avatar.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from "styled-components";
+import PropTypes from 'prop-types';
+import ScreenReaderContent from './shared/ScreenReaderContent';
+import * as tokens from "./shared/tokens.js";
+
+const avatarSizes = {
+  large: tokens.size_avatar_l,
+  medium: tokens.size_avatar_m,
+  small: tokens.size_avatar_s,
+  tiny: tokens.size_avatar_xs,
+};
+
+const Image = styled.div`
+  position: relative;
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: center;
+  border-radius: 100%;
+  height: ${(props) => `${props.avatarSize}px`};
+  width: ${(props) => `${props.avatarSize}px`};
+  line-height: ${(props) => `${props.avatarSize}px`};
+  background-image: url(${(props) => props.imageSrc});
+  background-size: cover;
+  background-position: 50%;
+  background-color: var(--cc_color_background_3);
+  border: ${(props) => props.avatarColor ? `2px solid ${props.avatarColor}` : null };
+  color: var(--cc_color_text_subtle);
+`;
+
+const Initials = styled.div`
+  font-size: ${(props) => `${props.textSize}px`};
+  font-weight: 700;
+  text-align: center;
+  font-family: 'Averta', system-ui, sans-serif;
+  text-transform: uppercase;
+`;
+
+const AvatarInitials = ({ initials, avatarSize }) => {
+  const textSize = Math.round(avatarSize / 2.5);
+  return <Initials textSize={textSize}>{initials}</Initials>;
+};
+
+const Avatar = ({
+  name,
+  src,
+  size,
+  color,
+  iconName,
+  customInitials,
+  className,
+  style,
+}) => {
+  const avatarSize = typeof size === 'number' ? size : avatarSizes[size];
+  const initials = name ? name.split(' ').map(x => x.charAt(0)).join('').substr(0, 3).toUpperCase() : null
+  return (
+    <Image
+      imageSrc={src}
+      avatarSize={avatarSize}
+      aria-label={name}
```

Comment:
As long as we have a name, I think we're rendering ScreenReaderContent, so you probably don't need this.

Reply:
The `aria-label` is there so that when we add a `hint` tooltip class it'll display properly, since the `hint` tooltip library uses `aria-label` to display the text in the tooltip.